keylime/src/structures/capabilities_negotiation.rs

```diff
@@ -90,34 +108,35 @@ pub struct EvidenceRequested {
     pub evidence_type: String,
     #[serde(skip_serializing_if = "Option::is_none")]
     pub chosen_parameters: Option<ChosenParameters>,
-    #[serde(skip_serializing_if = "Option::is_none")]
-    pub version: Option<String>,
 }
 
 #[derive(Serialize, Deserialize, Debug, Clone)]
 pub struct TpmParameters {
```

Comment:
To make it coherent with the  other structures, maybe this should be called `CertificationParameters`